src/report_aggregator_impl.cc

```diff
@@ -30,10 +30,10 @@ namespace google {
 namespace service_control_client {
 namespace {
 
-// A report can carry many operations, merge multiple report requests
-// into one report request until its number of operations reach this limit.
-// Each report is about 4KB now. Maximum allowed size from server is 1MB.
-const int kMaxOperationsToSend = 100;
+// Service control server limits each report data size to 1MB.
+// Roughly, each operation may have up to 50KB based on maximum of 100
+// aggregated logEntries (each log entry is about 500 bytes).
```

Comment:
The data size for all metrics is around 5KB.  If there are many log entries, most of data is from log entries.